### PR TITLE
Day02 update

### PR DESCRIPTION
**Pull request by ssingal05**

### CODE REVIEW

File: day00/README.md [hunk 1]
```diff
@@ -28,6 +28,6 @@ IDE, that's fine. Here's how to install IntelliJ:
 - `./bin/idea.sh  # this runs IntelliJ`
 - Set up IntelliJ according to your preferences!
 
-## Getting started with the code
+# Play around with IntelliJ
 
-Modify `ListClientExample` to use an ArrayList instead of a LinkedList.
```
Comment (davidabrahams) [inline day00/README.md]: Why do we not want them to do this anymore?

File: day00/README.md [hunk 2]
```diff
@@ -28,6 +28,6 @@ IDE, that's fine. Here's how to install IntelliJ:
...
+Try to make and execute some classes, create some unit tests, and try to get acclimated with the environment. If you have any questions, make sure to email the instructors!
```
Comment (davidabrahams) [inline day00/README.md]: This is too vague. They won't have any idea what theyre supposed to do

File: day02/README.md [hunk 3]
```diff
@@ -1,13 +1,69 @@
...
+    // QUESTIONS 1-7. COMMENT THIS LINE FOR QUESTION 8
+    MyLinkedList chickens;
+    
+    // QUESTION 8. COMMENT THIS LINE FOR QUESTIONS 1-7
+    //MyLinkedList<Chicken> chickens;
+    ...
+    // QUESTIONS 1-7. COMMENT THIS LINE FOR QUESTION 8
+    chickens = new MyLinkedList();
+    
+    // QUESTION 8. COMMENT THIS LINE FOR QUESTIONS 1-7
```
Comment (davidabrahams) [inline day02/README.md]: There is no question 8?

File: day02/solutions/MyLinkedListTest.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MyLinkedListTest {
+
+	// QUESTIONS 1-7. COMMENT THIS LINE FOR QUESTION 8
```
Comment (davidabrahams) [inline day02/solutions/MyLinkedListTest.java]: again, no question 8

File: day02/src/MyLinkedListTest.java [hunk 5]
```diff
@@ -1,25 +1,65 @@
...
-		list = new ArrayList<Integer>();
-		list.add(1);
-		list.add(2);
-		list.add(3);
-
-		mylist = new MyLinkedList<Integer>();
-		mylist.addAll(list);
+	public void setUp() throws Exception{
+
+		// QUESTIONS 1-7. COMMENT THIS LINE FOR QUESTION 8
```
Comment (davidabrahams) [inline day02/src/MyLinkedListTest.java]: here too

File: day00/README.md (hunk 2 above)
Comment (thecardkid) [inline day00/README.md]: Why is this file here?

File: day00/README.md (hunk 1 above)
Comment (thecardkid) [inline day00/README.md]: Cause we decided it was dumb I think. We could just have them write Hello World.

File: day00/build.xml [hunk 6]
```diff
@@ -1,63 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (thecardkid) [inline day00/build.xml]: Why is this file here?
